developers/weaviate/starter-guides/custom-vectors.mdx

```diff
@@ -8,187 +8,207 @@ image: og/docs/tutorials.jpg
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
-import Py4Code from '!!raw-loader!/_includes/code/starter-guides/custom-vectors-py4.py';
-import Py3Code from '!!raw-loader!/_includes/code/starter-guides/custom-vectors-py3.py';
-import Ts3Code from '!!raw-loader!/_includes/code/starter-guides/custom-vectors-ts3.ts';
-import Ts2Code from '!!raw-loader!/_includes/code/starter-guides/custom-vectors-ts2.ts';
+import ByovAllPyCode from '!!raw-loader!/_includes/code/quickstart.byov.all.py';
+import ByovAllPy3Code from '!!raw-loader!/_includes/code/quickstart.byov.all_v3.py';
+import ByovAllTsCode from '!!raw-loader!/_includes/code/quickstart.byov.all.ts';
+import ByovAllTsCodeLegacy from '!!raw-loader!/_includes/code/quickstart.byov.all.legacy.ts';
+import ByovAllShCode from '!!raw-loader!/_includes/code/quickstart.byov.all.sh';
 
 Weaviate is a vector database. [Vector databases](/blog/vector-library-vs-vector-database) store data objects and vectors that represent those objects. When you import a data object, you normally include a vector representation of that object as well. The vector representation is also called an "embedding."
```

Comment:
> When you import a data object, you normally include a vector representation of that object as well.

This is confusing to me, as to what we mean by "normally". It almost sounds as though the vectorizer integration is somehow abnormal.